_posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md

```diff
@@ -0,0 +1,102 @@
+These questions swirled through our minds:
+- What would help us in our everyday office life?
+- Which topic are on our I-always-wanted-to-work-on-that list?
+- Which topics are great but didn't make it yet and might never make it to one of the teams' sprints?
+
+My team member Birgit and I also brainstormed and added a topic to the list: Improving the [About us page](/about/) of the DevPortal.
+
+## Two weeks ago
+
+All topics were set and added to the list.
+Now, it was time to decide which topic we'd like to join.
+Let me tell you, it was a tough decision.
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
+
+These questions helped me to decide which topic I'd like to join:
+- How can I support the team that is working on that topic?
+- What will I learn when participating?
+- Which team members will I have? 
+
+Especially the last point was one of the most important ones for me.
+A Hackathon is the perfect chance to work together with colleagues that I don't work with on a daily basis.
+Maybe even with colleagues from different locations.
+This way, we can even build little "feature teams" for the two days.
+
+As Birgit covered the About us page, I took the chance to join a group which had the goal to categorize our merchants and their shops.
+The results of such a categorization could help Product Management to make decisions on future features or improvements.
+And of course, I also had my personal goal: Get some improvement ideas for our help articles.
+
+## Last Wednesday
+
+The time has come.
+It's Hackathon!
+And what could be the best way to start the first day?
+Right - Breakfast! 
+
+{% twitter https://twitter.com/epagesdevs/status/1110812533349322752 %}
+
+The room was packed with ePagees.
+Our colleagues from Jena, Barcelona, and Bilbao already arrived the day before and everyone was chatting... and eating.
+Around 10 am the last buns and bagels disappeared, and our CTO officially kicked off the event.
+
+During the rest of the day, everyone was diving into their topics, brainstorming, discussing, hacking, and prototyping.
+When walking across the floor, I've seen many colored white boards and concentrated faces.
+
+Around 6 pm one word was spread across the floor... PIZZA.
+In no time, all teams came together and enjoyed the food.
+While eating, I could already notice that many colleagues fell in love with their topics.
+They were quite satisfied with their results and had big plans for the next day... or even the night.
+
+{% twitter https://twitter.com/jensfischerhh/status/1110984432444456961 %}
+
+## Last Thursday
+
+Did I mention that we'll get Amazon vouchers for the first three groups?
```

Comment:
Did I mention that the winners of the hackathon get an Amazon voucher (and also the 2nd and 3rd place)?